src/components/food-block/FoodBlock.jsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import "../food-block/FoodBlock.css";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faStar } from "@fortawesome/free-solid-svg-icons";
+import { faHeart } from "@fortawesome/free-solid-svg-icons";
+
+function FoodBlock({ image, name }) {
```

Comment:
send only item as props and destructure keys needed
```suggestion
function FoodBlock({ foodItem }) {
    const {image, name, rate, image, time, price} = foodItem
}
```